Wikipedia/Code/ArticleCacheDBWriter.swift

```diff
@@ -68,7 +68,9 @@ final class ArticleCacheDBWriter: ArticleCacheResourceDBWriting {
                 
                 //append mobile-html-offline-resource URLRequests
                 for url in urls.offlineResourcesURLs {
-                    guard let urlRequest = self.articleFetcher.urlRequest(from: url) else {
+                    // We don't know or care what Content-Type these are
+                    let acceptWhateverHeaders = ["Accept": "*/*"]
```

Comment:
Super minor: may be worth naming this variable either `acceptAnyType` (to match the [specific directive](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Accept) itself) or just `headers` (to [match here](https://github.com/wikimedia/wikipedia-ios/blob/d416b8378e0977c95b1d643ee2dd00a09f99f34a/WMF%20Framework/ImageCacheController.swift#L99) and elsewhere in this PR) for clarity and consistency.

Reply:
Good idea, done! It might be worth consolidating these eventually, but for now hopefully the naming clarification helps.